x-pack/legacy/plugins/ml/public/data_frame/pages/data_frame_new_pivot/components/source_index_preview/source_index_preview.tsx

```diff
@@ -33,7 +33,7 @@ import {
   MlInMemoryTable,
   SortingPropType,
   SORT_DIRECTION,
-} from '../../../../../../common/types/eui/in_memory_table';
+} from '../../../../../components/ml_in_memory_table';
```

Comment:
Can you check if this table works in the Transforms wizard. I am getting errors that the `pageIndex` property is missing from the `pagination` part of the `EuiBasicTable` so the tables fail to load, with this error seen in the console.

![image](https://user-images.githubusercontent.com/7405507/65160931-96828e00-da2e-11e9-85ef-1eaef293885f.png)

Reply:
Good catch! Fixed with d981ba0a0f7c62b781cc099bff9ef1c2fa58d7c7